assets/hmbkp.js

```diff
@@ -107,9 +107,17 @@ jQuery(document).ready(function ( $ ) {
             }
         }
 
-    });
+    } );
+
+    // Closing ThickBox Modal Window 
+    $( document ).on( 'click', '.hmbkp-thickbox-close', function ( e ) {
+
+        e.preventDefault();
+        tb_remove();
```

Comment:
One way to fix this is to replace it with `window.parent.tb_remove();`

Do you want me to proceed?